content/guides/models/track/log/distributed-training.md

```diff
@@ -85,7 +93,96 @@ Explore the W&B App UI to view an [example dashboard](https://wandb.ai/ayush-tha
+
+{{% alert %}}
+Consider using an environment variable to set the run ID of the primary node that you can then define in each worker node's machine.
+{{% /alert %}}
+
+The following sample code demonstrates the high level requirements for tracking multiple processes to a single run:
+
+```python
+import wandb
+
+# Initialize a run in the primary node
+run = wandb.init(
+    entity="entity",
+    project="project",
+	settings=wandb.Settings(x_label="rank_0", mode="shared", x_primary=True)
+)
+
+# Note the run ID of the primary node.
+# Each worker node needs this run ID.
+run_id = run.id
+
+# Initialize a run in a worker node using the run ID of the primary node
+run = wandb.init(
+	settings=wandb.Settings(x_label="rank_1", mode="shared", x_primary=False),
+	id=run_id,
+)
+
+# Initialize a run in a worker node using the run ID of the primary node
+run = wandb.init(
+	settings=wandb.Settings(x_label="rank_2", mode="shared", x_primary=True),
+	id=run_id,
+)
+```
+
+In a real world example, each worker node might be on a separate machine.
+
+{{% alert %}}
+See the [Distributed Training with Shared Mode](https://wandb.ai/dimaduev/simple-cnn-ddp/reports/Distributed-Training-with-Shared-Mode--VmlldzoxMTI0NTE1NA) report for an end-to-end example on how to train a model on a multi-node and multi-GPU Kubernetes cluster in GKE.
+{{% /alert %}}
+
+View console logs from multi node processes in the project that the run logs to:
+
+1. Navigate to the project that contains the run.
+2. Click on the **Runs** tab in the left sidebar.
+3. Click on the run you want to view.
+4. Click on the **Logs** tab in the left sidebar.
+
+You can filter console logs based on the labels you provide for `x_label` in the UI search bar located at the top of the console log page. For example, the following image shows which options are available to filter the console log by if values  `rank0`, `rank1`, `rank2`, `rank3`, `rank4`, `rank5`, and `rank6` are provided to `x_label`.` 
+
+{{< img src="/images/track/multi_node_console_logs.png" alt="Console logs from multiple nodes with the x_label filter applied." >}}
+
+W&B aggregates system metrics from all nodes and displays them in the W&B App UI. For example, the following image shows a sample dashboard with system metrics from multiple nodes. Each node possesses a unique label (`rank_0`, `rank_1`, `rank_2`) that you specify in the `x_label` parameter.
+
+{{< img src="/images/track/multi_node_system_metrics.png" alt="Line plot panel with numerous semi linear lines depicting system metrics logged by a multi node process." >}}
+
+See [Line plots]({{< relref "/guides/models/app/features/panels/line-plot/" >}}) for information on how to customize line plot panels. 
+
+## Avoid common distributed training issues with W&B Service
```

Comment:
This might be better as "Troubleshooting" or "Antipatterns" or something more terse. It seems like it should go after the Examples below.